org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/typesystem/conformance/TypeConformanceComputationArgument.java

```diff
@@ -13,6 +13,9 @@
  */
 public class TypeConformanceComputationArgument {
 
+	public static final TypeConformanceComputationArgument RAW = new TypeConformanceComputationArgument(true, false, true, true, false, true);
+	public static final TypeConformanceComputationArgument DEFAULT = new TypeConformanceComputationArgument();
+	
```

Comment:
should the constructor with no arguments be private?

Reply:
I considered that, too but was a little afraid that clients may use it already in their code.

Reply:
good point. maybe mark it as deprecated?

Reply:
:+1: Done